share/picongpu/examples/Bunch/etc/picongpu/32.cfg

```diff
@@ -66,7 +66,10 @@ TBG_plugins="!TBG_eBin                     \
 
 TBG_deviceDist="!TBG_devices_x !TBG_devices_y !TBG_devices_z"
 
-TBG_programParams="-d !TBG_deviceDist \
+TBG_programParams="--openPMD.period 1000 \
```

Comment:
Please remove this changes, there was no dumping enabled before so there is not need to change it.